### ISSUE TITLE
company-web-html backend memory leak

### ISSUE THREAD

**Issue by axyz** [syl20bnr/spacemacs#3263]
I've noticed on web mode that after a while of working (above all adding lot of parenthesis, tags and quotes) a memory leak seems to happen and when all the ram is full emacs crashes (or it becomes slow as hell I can guess if you have some high swappiness level)

I've tried disabling a lot of things, but at the end the problem seems to stop only removing `company-web-html` from the `company-backends` list.

As a temporary fix that backend could be disabled, but first let's try to see if this happen only on my configuration or if it is a widespread problem.

**Comment by robbyoconnor** [comment]
I wonder if this should be reported upstream? We don't do anything special, so it's not a spacemacs issue. Link to the issue from the upstream report.

**Comment by robbyoconnor** [comment]
As a temporary fix
```
(setq dotspacemacs-configuration-layers 
 '((auto-completion :disabled-for html)))
```

**Comment by axyz** [comment]
That's probably the case 

I just want to check if the problem exist for other people as well, in that case we may need to apply the temporary fix on master I guess (I cannot work in web-mode  without having frequent crashes) 

 In the meanwhile I'm going to report it on company repository. 

Thanks for the fix.

**Comment by robbyoconnor** [comment]
It'd be fixed in `develop` before it hits `master`.

**Comment by robbyoconnor** [comment]
Also, in the future when reporting bugs: 

1. `SPC hds` should always be included in bug reports.
2. Check on `develop` to see if the bug is fixed. If it is, it will be released soonish
3. If those fail, then report it.

**Comment by axyz** [comment]
I'm on develop of course. I've just thought that if it is a backend problem it may hit master people as well who expect a stable environment. 

Going to include `sec hds` when at home,  thanks.

**Comment by osv** [comment]
Hi, I'm developer of company-web. Memory issue was fixed in version v2.0 of company-web few month ago. I think you can recheck with your environment. Bumping this thread because spacemacs have 1700 open issues and this one probable can be closed :)
